apps/communications/contacts/js/views/form.js

```diff
@@ -369,6 +371,36 @@ contacts.Form = (function() {
     };
   };
 
+  function showForm(event) {
+    if(event.key === 'Control') {
```

Comment:
I've checked UX requirements in section 4.6 - it seems that you don't need softkeys at all, this 'form' is expected to be shown as result of pressing 'Back/Clear' button, not a softkey.